source/wp-content/themes/wporg-developer-2023/src/search-filters/index.php

```diff
@@ -43,20 +43,26 @@ function render( $attributes, $content, $block ) {
 		'wp-parser-method'   => __( 'Methods', 'wporg' ),
 	);
 	$qv_post_type = array_filter( (array) get_query_var( 'post_type' ) );
-	$no_filters   = $GLOBALS['wp_query']->is_empty_post_type_search;
+	$no_filters   = true === $GLOBALS['wp_query']->is_empty_post_type_search;
 	if ( in_array( 'any', $qv_post_type ) || $no_filters ) {
 		// No filters used.
 		$qv_post_type = array();
 	}
 
-	$content = sprintf( '<span> %s</span>', __( 'Filter by type:', 'wporg' ) );
+	$content = '<div>';
+	$content .= sprintf(
+		'<button id="wp-block-wporg-search-filters-all" aria-pressed="%1$s">%2$s</button>',
+		empty( $qv_post_type ) ? 'true' : 'false',
+		__( 'All', 'wporg' ),
+	);
+	$content .= '</div>';
 
 	foreach ( $search_post_types as $post_type => $label ) {
 		$input_id = esc_attr( $post_type );
 		$checked = checked( in_array( $post_type, $qv_post_type ), true, false );
 		$content .= '<div>';
 		$content .= sprintf( '<input id="%1$s" type="checkbox" name="post_type[]" value="%1$s" %2$s />', $input_id, $checked );
-		$content .= sprintf( '<label for="%s">%s</label>', $input_id, $label );
+		$content .= sprintf( '<label for="%s" tabindex="0">%s</label>', $input_id, $label );
```

Comment:
When I listen to this in VoiceOver, I'm told that I'm on a text element, so there's no indication that I can interact with this.

I think you want the checkbox to be visible to be in the tab order, and then you can use a sibling selector to style the label when an input is focused. That's the approach I used [in the query filters](https://github.com/WordPress/wporg-mu-plugins/blob/f6f13cf34e308c2c5553676909c61f1d8b7c271c/mu-plugins/blocks/query-filter/postcss/style.pcss#L128), at least.

Reply:
Yeah that's much better, [updated](https://github.com/WordPress/wporg-developer/pull/292/commits/8f015a3992f944554b6654ed1791e132a93a7dfb), thanks